Source/EasyNetQ/Consumer/AsyncBasicConsumer.cs

```diff
@@ -53,12 +56,14 @@ public override async Task OnCancel(params string[] consumerTags)
         if (logger.IsInfoEnabled())
         {
             logger.InfoFormat(
-                "Consumer with consumerTags {consumerTags} has cancelled",
-                string.Join(", ", consumerTags)
+                "Consumer {consumerId} cancelled for consumerTags {@consumerTags}",
```

Comment:
@Pliner I suggest to remove all `@` from format strings. For arrays and dictionaries `@` is especially pointless if you keep Serilog in mind. Serilog supports them OOTB. MEL too by the way.